crypto/rand/rand_lib.c

```diff
@@ -152,7 +152,7 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
     } else {
         pool = rand_pool_new(entropy, min_len, max_len);
         if (pool == NULL)
-            return 0;
+            goto err;
```

Comment:
Any reason for this change?

Reply:
The is no other reference to the label err when the conditionally included block is absent.  This provides one.  This seemed tidier than conditionally defining the label too.

Without something the build fails with a _label defined but not used_ message.

Reply:
Depending on what happens with #8660 it might be possible to revert this change.

Reply:
Note that #8660 currently depends on this.  Prediction resistance is required to seed  a strong DRBG from a weaker one.

Reply:
This went away with later changes.